modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosController.scala

```diff
@@ -131,6 +130,20 @@ object GmosController {
         Enumerated.of(InBeam, OutOfBeam)
     }
 
+    sealed trait ElectronicOffset extends Product with Serializable
+
+    object ElectronicOffset {
+      case object ElectronicOffsetOn extends ElectronicOffset
+      case object ElectronicOffsetOff extends ElectronicOffset
```

Comment:
I think `On` and `Off` are enough. In every use of those values it is clear that they refer ro `ElectronicOffset`.